docs/reference/cryptography.rst

```diff
@@ -0,0 +1,28 @@
+============
+Cryptography
+============
+
+Transport Layer Security (TLS)
+------------------------------
+All network endpoints exposed by MicroOVN services are secured using multiple
+components of the `TLS protocol`_, including encryption, authentication and
+integrity.  Through the use of the `Ubuntu OpenSSL`_ packages, TLS versions
+below 1.2 are disabled for security reasons.
+
+There are two self-signed certificate authorities in use, one for the
+`MicroCluster`_ based ``microovnd`` daemon, another for the `OVN`_ daemons.
+These are initialised during the initial bootstrap of the cluster.
+
+Keys are generated using a 384 bit Elliptic Curve algorithm often referred to
```

Comment:
would it be worth linking to further information on this algorithm somewhere?

Reply:
Yeah, we probably should. At first attempt the best reference I found looked a bit outdated, which made me unsure, I think we need something better than Wikipedia.

I'll make another attempt at finding a more recent reference.

Reply:
It's at least referred to in RFC 5480, let's try that as a reference.

Reply:
@crypticC0der pointed to the wikipedia page on [elliptic curve cryptography](https://en.wikipedia.org/wiki/Elliptic-curve_cryptography), while generic it is quite decent, let's use it!